fs/src/lib.rs

```diff
@@ -113,8 +123,16 @@ impl FileSystemProvider {
     ) -> Result<Vec<u8>, Box<dyn Error + Sync + Send>> {
         let blob = sanitize_blob(&blob);
         let bfile = self.blob_to_path(&blob);
-        std::fs::remove_file(&bfile)?;
-        Ok(vec![])
+        serialize(match std::fs::remove_file(&bfile) {
```

Comment:
I've just been doing a refactor to code like this that might make this look "more idiomatic".

You can replace a match that returns 2 values of the same type with a `.map_or_else` function, where the first closure is what you do with the error value and the second closure is what you do with the Ok value.

From one of my actors undergoing this refactor:
```rust
kv::add_room(id, description).map_or_else(
        |e| RoomAck::fail(&format!("{}", e)),
        |_v| RoomAck::success(),
    )
```

Reply:
Good call, I'll look at refactoring this 😃